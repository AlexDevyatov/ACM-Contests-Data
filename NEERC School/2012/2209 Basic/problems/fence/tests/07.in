3
-100000000 -100000000
100000000 100000000
-100000000 100000000
99999998 99999999
