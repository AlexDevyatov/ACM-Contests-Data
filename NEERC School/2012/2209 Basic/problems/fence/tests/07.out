4
-100000000 -100000000
99999998 99999999
100000000 100000000
-100000000 100000000
